### DISCUSSION TITLE
Should the MenuItem stay in the hovered state when the cursor moves off of it?

### DISCUSSION THREAD

**Discussion by sbdchd** [adobe/react-spectrum#5727] [Q&A] [answered]
With a react-aria button, if you move your cursor off of it the hover state goes away, but with the MenuItem, it seems like it gets stuck

https://github.com/adobe/react-spectrum/assets/7340772/3afe9b22-ca7a-4743-8635-ac35d19495eb

**Comment by reidbarber** [comment, marked as answer]
This will get fixed in https://github.com/adobe/react-spectrum/pull/5648 (Related comment: https://github.com/adobe/react-spectrum/pull/5648/files#r1459716021)

**Reply by laurenmbeatty** [reply]
Hi, @reidbarber I see that PR was merged, but will the fix for this get bundled in a different release? I'm still seeing the same behavior on the [docs page](https://react-spectrum.adobe.com/react-aria/Menu.html) and in my own code using the `latest`. Thanks!

**Reply by reidbarber** [reply]
@laurenmbeatty In that docs example, the menu item style is applied on focus, not hover. Hovering the item focuses it, but moving your pointer away doesn't necessarily remove focus (until something else is clicked or another item is hovered). If you inspect the element, you should see that the `data-hovered` state is properly updated when your pointer leaves the item.

**Reply by laurenmbeatty** [reply]
Oh, yes...thank you so much!
